### ISSUE TITLE
LastUpdatedTimeRaw not being populated

### ISSUE THREAD

**Issue by DanTurco** [IEvangelist/azure-cosmos-dotnet-repository#357]
**Describe the bug**
When creating or updating a record the LastUpdatedTimeRaw property is never set. Even though the update and creates are successful that value always is being set to zero, even though in the actual cosmos database it has a valid value.

**To Reproduce**
Make call to (CreateAsync, UpdateAsync) on an IRepository instance where the class to save inherits from TimeStampedItem.

```csharp
public class BaseEntity : TimeStampedItem, IItemWithEtag, IDocument
```

**Expected behavior**
The LastUpdatedTimeRaw with the ["_ts"] json attribute is populated and returned in the response from the CosmosClient.

**Actual behavior**
LastUpdatedTimeRaw is returned as 0. and the getter property LastUpdatedTimeUtc defaults to min date.

**Environment summary**
SDK Version: 7.1.0
OS Version: Windows
Frameword: .NET7

**Additional context**
Repo setup
```csharp
builder.Services.AddCosmosRepository(
        options =>
        {
            options.IsAutoResourceCreationIfNotExistsEnabled = builder.Environment.IsDevelopment();
            options.CosmosConnectionString = builder.Configuration.GetValue<string>("Connections:Store:EndpointUrl");
            options.ContainerPerItemType = true;
            options.OptimizeBandwidth = false; //we set false to use the etag for concurrency checks
            //options.ContainerId = builder.Configuration.GetValue<string>("Connections:Store:CollectionName");
            options.DatabaseId = builder.Configuration.GetValue<string>("Connections:Store:DatabaseName");
            options.ContainerBuilder.Configure<ExportsAPI.Domain.Entities.Client>(userContainerOptions =>
            {
                userContainerOptions.WithContainer(builder.Configuration.GetValue<string>("Connections:Store:CollectionName"));
            });
        });
```
.net code after success Create
![20 125 209 179 2023-10-19 11-59-48](https://github.com/IEvangelist/azure-cosmos-dotnet-repository/assets/147526328/788624b3-3ad1-42c9-836e-6bd05cfdff04)

database
![20 125 209 179 2023-10-19 12-00-33](https://github.com/IEvangelist/azure-cosmos-dotnet-repository/assets/147526328/42a9aa36-5f0e-4008-95de-cc9c8fd4982b)

.net code after success Update
![20 125 209 179 2023-10-19 12-02-30](https://github.com/IEvangelist/azure-cosmos-dotnet-repository/assets/147526328/502063ed-c979-4267-b37e-354c92ead490)

database
![20 125 209 179 2023-10-19 12-03-06](https://github.com/IEvangelist/azure-cosmos-dotnet-repository/assets/147526328/fc2f78b5-f39e-464a-8fd1-594a95e218cf)

**Comment by DanTurco** [comment]
It looks like the bug is in the TimeStampedItem for the property 

 [JsonProperty("_ts")]
  public long LastUpdatedTimeRaw { get; }

it needs to have a setter for the serialization to work correctly.

My fix was to just copy this class and interface into my project and make the fix.. Times now populate properly

**Comment by mumby0168** [comment]
Yeah that looks to be correct @DanTurco. Would you be interested in applying the fix into this library as well so overs can benefit from the the bug fix?

**Comment by sven5** [comment]
This seems to have changed to

```
[JsonProperty("_ts")]
public long LastUpdatedTimeRaw { get; private set; }
```

Does anyone know which purpose does the private setter have?
